sdks/python/apache_beam/testing/test_stream.py

```diff
@@ -183,9 +182,20 @@ class TestStream(PTransform):
   Each event emits elements, advances the watermark or advances the processing
   time. After all of the specified elements are emitted, ceases to produce
   output.
+
+  When expanded, if only the default output or only one output tag is specified,
+  then a PCollection will be returned. Otherwise, a dictionary of output names
+  to PCollection will be returned.
```

Comment:
```suggestion
  Applying the PTransform will return a single PCollection if only the default output or only one output tag has been used. Otherwise a dictionary of output names to PCollections will be returned.
```